src/status_im/commands/handlers/loading.cljs

```diff
@@ -82,12 +83,15 @@
     (status/parse-jail
      whisper-identity (str local-storage-js ethereum-id-js file)
      (fn [result]
-       (let [{:keys [error result]} (types/json->clj result)]
-         (log/debug "Parsing commands results: " error result)
+       (let [{:keys [error result]} (types/json->clj result)
```

Comment:
Can we call the first result something else? A bit confusing when it shadows same result variable.

Also, why do we need to do json->clj twice now?

Reply:
I stuck a bit with figuring out how to create nested JSON with arbitrary fields on obj-c side and gave up because of time limits, thus just passed string and parsed it with another `json->clj`.  Yeah we can rename `result` though that's not something introduced by current PR, so i would skip this for another one.